modules/index/src/main/java/org/apache/ignite/internal/index/SortedIndex.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.index;
+
+import java.util.BitSet;
+import org.apache.ignite.internal.schema.BinaryTuple;
+import org.apache.ignite.internal.util.Cursor;
+
+/**
+ * An object describing a sorted index.
+ *
+ * <p>Extends a basic index with operations related to sorted structures.
+ */
+public interface SortedIndex extends Index {
```

Comment:
Just pure interface for now ? Do we need some kind of TODO ? here ?

Reply:
Why do we need a TODO here?

Reply:
And why do we need pure interface?

Reply:
Well, actually it's OK to commit an interface without any implementation if you need to provide a contract. But this not the case. So removed it